LiskHQ/lisk-service
# Add swagger documentation for API v3 - Closes #1184

### What was the problem?
This PR resolves #1184 

### How was it solved?

- [x] Update SDK version to latest canary
- [x] Swagger specs is available through HTTP
- [x] Specs file is compatible with online tools

### How was it tested?
- Run gateway and hit endpoint `http://localhost:9901/api/v3/spec`
- Copy JSON and use swagger editor to verify the docs
